vnext/JSI/Shared/ChakraRuntime.h

```diff
@@ -328,7 +347,99 @@ class ChakraRuntime : public facebook::jsi::Runtime {
   bool evaluateSerializedScript(
       const facebook::jsi::Buffer &scriptBuffer,
       const facebook::jsi::Buffer &serializedScriptBuffer,
-      const std::string &sourceURL);
+      const std::string &sourceURL,
+      JsValueRef *result);
+
+  enum class PropertyAttibutes {
```

Comment:
>PropertyAttibutes [](start = 13, length = 17)

(nit) maybe PropertyFlags? Usually such bit structures are called flags.

Reply:
I do not think the term 'Flags' is a better term here than 'Attributes'. I used the 'PropertyAttributes' based on the JSC API: [https://docs.microsoft.com/en-us/dotnet/api/javascriptcore.jspropertyattributes?view=xamarin-ios-sdk-12](https://docs.microsoft.com/en-us/dotnet/api/javascriptcore.jspropertyattributes?view=xamarin-ios-sdk-12)

---
In reply to: [470241593](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470241593) [](ancestors = 470241593)